pingen/README.rst

```diff
@@ -0,0 +1,108 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
+
+===========================
+Integration with pingen.com
+===========================
+
+What is pingen.com
+==================
+
+Pingen.com is a paid online service.
+It sends uploaded documents by letter post.
+
+Scope of the integration
+========================
+
+One can decide, per document / attachment, if it should be pushed
+to pingen.com. The documents are pushed asynchronously.
+
+A second cron updates the informations of the documents from pingen.com, so we
+know which of them have been sent.
+
+Configuration
+=============
+
+The authentication token is configured on the company's view. You can also
+tick a checkbox if the staging environment (https://stage-api.pingen.com)
+should be used.
+
+The setup of the 2 crons can be changed as well:
+
+ * Run Pingen Document Push
+ * Run Pingen Document Update
```

Comment:
Can you please update this part as well? Explaining how to configure the webhooks and so on?